backend/fms_core/serializers.py

```diff
@@ -256,13 +241,20 @@ def get_depleted(self, obj):
 
 class SampleSerializer(serializers.ModelSerializer):
     extracted_from = serializers.SerializerMethodField()
+    sample_kind = serializers.CharField(read_only=True, source="derived_sample_not_pool.sample_kind.id")
```

Comment:
We need to have a clear idea how we'll have this working once we have pool or we may very well have to redesign the whole thing again ... From our previous discussion i think the word is to hide from list all pool data that is not from sample so this might actually work long term. Lets think it over some.